docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,60 @@
+# Deploy the GitHub runner image builder for the first time
+
+## What you'll do
+
+- Deploy the charm.
+- Integrate with GitHub runners.
+
+## Requirements
+
+- A running instance of [OpenStack](https://microstack.run/docs/single-node).
+
+## Steps
+
+- Deploy the [GitHub runner charm in OpenStack mode](https://charmhub.io/github-runner/docs/how-to-openstack-runner).
+
+- Deploy the GitHub runner image builder charm. For information on OpenStack credentials, refer 
+to the official [OpenStack documentation](https://docs.openstack.org/python-openstackclient/pike/configuration/index.html).
+
+```
+OPENSTACK_AUTH_URL=<openstack-auth-url, e.g. http://my-openstack-deployment/openstack-keystone>
+OPENSTACK_PASSWORD=<openstack project password>
+OPENSTACK_PROJECT_DOMAIN_NAME=<openstack project domain name>
+OPENSTACK_PROJECT_NAME=<openstack project name>
+OPENSTACK_USER_DOMAIN_NAME=<openstack user domain name>
+OPENSTACK_USERNAME=<openstack username>
+juju deploy github-runner-image-builder \
+--config openstack-auth-url=$OPENSTACK_AUTH_URL \
+--config openstack-password=$OPENSTACK_PASSWORD \
+--config openstack-project-domain-name=$OPENSTACK_PROJECT_DOMAIN_NAME \
+--config openstack-project-name=$OPENSTACK_PROJECT_NAME \
+--config openstack-user-domain-name=$OPENSTACK_USER_DOMAIN_NAME \
+--config openstack-user-name=$OPENSTACK_USERNAME
+```
+
+- Verify that the image is being built via juju logs:
+```
+juju debug-log --include=github-runner-image-builder/0
+```
+
+- Verify that the image is successfully built. 
+```
+openstack image list | grep noble-x64
+```
+
+- Integrate with GitHub runners. 
+```
+juju relate github-runner-image-builder github-runner
```

Comment:
Nitpick: Should we be using `juju integrate` instead? I seem to recall that `juju relate` is being deprecated